NOTICE.txt

```diff
@@ -449,6 +591,19 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ---
 
+## sass
+
+This product contains 'sass' by sass.
+
+Sass makes CSS fun again. Sass is an extension of CSS, adding nested rules, variables, mixins, selector inheritance, and more. It's translated to well-formatted, standard CSS using the command line tool or a plugin for your build system.
+
+* HOMEPAGE:
+  * https://github.com/sass/sass
+
+* LICENSE: ?
```

Comment:
No license?

Reply:
https://sass-lang.com/
MIT License at the bottom of the page.